lib/prefix.h

```diff
@@ -217,6 +236,8 @@ struct prefix_evpn {
 
 static inline int is_evpn_prefix_ipaddr_none(const struct prefix_evpn *evp)
 {
+	if (evp->prefix.route_type == 1)
```

Comment:
That is a interesting question. When it comes to EVPN route types a lot of people seem to prefer number (rt-1, rt-2, rt-3) instead of the generic name like EAD, MAC-IP (which is ambiguous and requires referring to the RFC). I can surely added a define that looks like this -
#define EVPN_RT_TYPE_1 1
#define EVPN_RT_TYPE_2 2

But I fail to see the value in that. Also this is existing code so if I clean up it has to be all not just the new route I added. I will reflect on it and do it in a separate PR.

Reply:
I have added a separate commit to the PR for this clean up. Let me know if that seems better.